_posts/2019-09-17-RasingResearchSoftware.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+image: '/img/Black_hole_-_Messier_87_crop_max_res.jpg'
+title: "Raising the Profile of Research Software: how this community action started"
```

Comment:
Perhaps add "Request For Comment:" or something to show we are requesting feedback.